include/syscalls.h

```diff
@@ -69,7 +69,7 @@
 	ID(signalMask) \
 	ID(signalSuspend) \
 	ID(priority) \
-	\
+\
```

Comment:
Why this change? Imho better to keep indentation level. Perhaps there are some objective arguments?

Reply:
Our sincerlys overloard clang-format' whims, I'll revert those